### PR TITLE
Content/wip/hardening changes

### PR DESCRIPTION
**Pull request by marybyrnepark**
Incorporating hardening changes for Apply & Track Claims

### CODE REVIEW

**Comment by marybyrnepark** [comment]
@peggygannon could you pls proofread this PR? 
I will direct you to what you need to be looking at in the issue. Thanks!

**Comment by marybyrnepark** [comment]
@peggygannon I incorporated changes into https://vetsgov-pr-7812.herokuapp.com/health-care/ (just had to change "application process" to "how to apply")
--also made some changes to simplify extraneous verbiage in https://vetsgov-pr-7812.herokuapp.com/employment/vocational-rehab-and-employment/family-members/
Can you take a look?

**Review by peggygannon — COMMENTED** [review]
@marybyrnepark Hi, Mary. Proofread for this PR is complete. I think the following pages need a further look to determine if we should tweak and/or delete one of the "how to apply" headers.

https://vetsgov-pr-7812.herokuapp.com/employment/vocational-rehab-and-employment/apply-vre/
--We have two "how to apply..? headers. Do we want to tweak or delete one of them? 
**BETH COMMENT:** @marybyrnepark To avoid two subheads starting essentially the same way, (how to apply / how do I apply), please change "How to apply if you already have your VA disability rating" to "If you already have your VA disability rating, take these steps to apply:"

https://vetsgov-pr-7812.herokuapp.com/employment/vocational-rehab-and-employment/family-members/
--we have header: How do I get these benefits? , then below that: "how do I apply?
do we want to tweak or delete one of them?
**BETH COMMENT:** Mary, you already solved this one by removing "How do I get these benefits?" and moving up "Follow these steps to apply for Chapter 36 services:" **DONE**

https://vetsgov-pr-7812.herokuapp.com/housing-assistance/home-loans/apply-for-certificate-of-eligibility/
we have header "How do I get my COE?, **BETH COMMENT:** Mary, please change to "How do I prepare before starting my application?" Then run the next two sentences together in plain text (no bold): "Gather the information you'll need to apply for your COE. Click on the description below that matches you best to find out what you'll need. (This is almost exactly what's on the page now, but without bolding, the addition of the word "below," and the removal of the word "information")
then we have header later down the page: "How do I apply for my COE?"
Do we want to delete one of these headers?

File: content/pages/pension/index.md [hunk 1]
```diff
@@ -1,40 +1,208 @@
 ---
```
Comment (phiden) [inline content/pages/pension/index.md]: This is the merger version of the page. Why is it in this PR?

**Review by phiden — COMMENTED** [review]
As soon as we figure out that merger page & the breadcrumbs, this is good to go

**Review by phiden — APPROVED** [review]

**Comment by bethpotts** [comment]
@phiden This is ready to merge.

**Review by bethpotts — APPROVED** [review]
This is ready to merge.

**Pull request merged**